### PR TITLE
replaces privateEnv with small util getEnvVar in lix/server

### PR DESCRIPTION
**Pull request by martin-lysk**
This one is blocked by #2562 
* [x] merge main in again and remove the `@inlang/env-variables` dependency as soon as #2562  is merged

### CODE REVIEW

**Comment by janfjohannes** [comment]
@martin-lysk  what is the status?

**Comment by martin-lysk** [comment]
@janfjohannes i removed the depedency from env-variables package - now its ready to review

File: lix/packages/server/src/util/getEnv.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { config } from "dotenv"
+
+// we configure the dotenv with default - which loads the .env file from the execution folder
+config()
+
+export function getEnvVar(varName: string, descirption: string, defaultValue?: string) {
+	const envVar = process.env[varName]
+	if (!envVar) {
```
Comment (janfjohannes) [inline lix/packages/server/src/util/getEnv.ts:8]: empty string needs to be a valid env variable.

File: lix/packages/server/src/util/getEnv.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { config } from "dotenv"
+
+// we configure the dotenv with default - which loads the .env file from the execution folder
+config()
+
+export function getEnvVar(varName: string, descirption: string, defaultValue?: string) {
```
Comment (janfjohannes) [inline lix/packages/server/src/util/getEnv.ts:6]: description should be optional

Comment (janfjohannes) [inline lix/packages/server/src/util/getEnv.ts:6]: better have call signature getEnvVar(varname, {default, description})   its more expandable and clear what the args are.

**Review by janfjohannes — CHANGES_REQUESTED** [review]
see comments

**Comment by janfjohannes** [comment]
@martin-lysk  as i need this PR i will take over.

**Review by janfjohannes — APPROVED** [review]

**Pull request merged**